modules/database/src/ioc/dbStatic/dbStaticLib.c

```diff
@@ -1381,6 +1381,11 @@ char * dbGetFieldName(DBENTRY *pdbentry)
     return(pflddes->name);
 }
 
+dbfType dbGetDbfFieldType(DBENTRY *pdbentry)
+{
+    return(pdbentry->pflddes->field_type);
```

Comment:
This code could result in a segfault if it gets called when no field is selected and `pdbentry->pflddes` is NULL – see the check for that possibility in the routines either side of it. The question then is what to return in that case, which is a problem because there is no really appropriate `dbfType` enum value for that. If you change the return type of your routine to be an `int` you can then return -1 in that case. Your application should be able to handle that value too.

One other request, could you make the routine name `dbGetFieldDbfType()` so it's closer to the other `dbGetField...()` routine names.

With those changes I'd be happy to accept this, thanks!

Reply:
Thank you for your review, I amended the commit with your suggested changes.
I did not check for NULL before, because I thought selecting the field was always done when you call any `dbGetField*()`. I guess there are exceptions to this.